### ISSUE TITLE
Audio loop

### ISSUE THREAD

**Issue by lightonflux** [supertuxkart/stk-code#1830]
I now have the second time heard a sound loop that doesn't sound like it is supposed to be played.

The first time i heard the drifting noise in a loop and now i hear the horns of the start sound.

Here a video WITH sound (because i had SSR already setup): http://videobin.org/+8ea/b87.ogg

And the log: http://paste.ubuntu.com/9565045/

**Comment by lightonflux** [comment]
I couldn't reproduce it with newer dev versions. Will update when it happens again.

**Comment by hiker** [comment]
I fixed a few threading related audio bugs - most likely this is fixed by those fixes. Please comment on this ticket if it happens again.

Thanks for the report!
  Joerg

**Comment by lightonflux** [comment]
The issue reappeared when we used the version from the gentoo games-overlay and played half an evening with it (3-5 hours). [ebuild file](https://github.com/hasufell/games-overlay/blob/master/games-action/supertuxkart/supertuxkart-0.9.ebuild). 

@hiker if no other fixes regarding this problem were applied between the release of 0.8.2 and 0.9 then the problem is still present. 

On my notebook with 0.9-4~bpo8+1 (Debian Jessie) the bug hasn't reappeared. Maybe the bug less likely to appear after the fixes you mentioned. I played less than 3h on my notebook so maybe i just didn't hit that bug yet.
